code/datums/traits/negative_quirk.dm

```diff
@@ -341,14 +341,18 @@
 	var/mob/living/carbon/human/H = quirk_holder
 	if(H.dna.species.id in list("shadow", "nightmare"))
 		return //we're tied with the dark, so we don't get scared of it; don't cleanse outright to avoid cheese
-	var/turf/T = get_turf(quirk_holder)
-	if(T.get_lumcount() <= 0.2)
-		if(quirk_holder.m_intent == MOVE_INTENT_RUN)
-			to_chat(quirk_holder, "<span class='warning'>Easy, easy, take it slow... you're in the dark...</span>")
-			quirk_holder.toggle_move_intent()
-		SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "nyctophobia", /datum/mood_event/nyctophobia)
-	else
+	var/turf/holder_turf = get_turf(quirk_holder)
+
+	var/lums = holder_turf.get_lumcount()
+
+	if(lums > LIGHTING_TILE_IS_DARK)
 		SEND_SIGNAL(quirk_holder, COMSIG_CLEAR_MOOD_EVENT, "nyctophobia")
```

Comment:
it would be nice if this set you back to running if it knew you were running before, you could store it on the quirk itself since afaik quirk instances are unique to the mob.

Also this shouldn't send the signal every tick as it currently does, it should send if the lumcount changes to a safe number and only then. you can cache the lastprocess lumcount on the quirk as well

Reply:
> it would be nice if this set you back to running if it knew you were running before,
Oh god yes please

Reply:
this was something I accidentely ported from the quirk rework, and so I have removed it

Reply:
What? What was an accidental port?

Reply:
> What? What was an accidental port?

this part was only intended to make the number 0.02 a define "LIGHTING_TILE_IS_DARK" as thats whats in the pr I linked